pydruid/client.py

```diff
@@ -402,8 +413,6 @@ def _post(self, query):
                     err = json.loads(e.read().decode("utf-8"))
                 except (ValueError, AttributeError, KeyError):
                     pass
-                else:
```

Comment:
We needed to debug a connection to a druid server and we could only see the error "Internal Server Error" (something like at anyway), but not the longer, explanatory error message. So we let the longer, explanatory error message in instead of going the extra mile of removing that (vital?) piece of information.